uitools/import/Esri/ArcGISRuntime/Toolkit/UtilityNetworkTrace.qml

```diff
@@ -0,0 +1,337 @@
+/*******************************************************************************
+ *  Copyright 2012-2022 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+import Esri.ArcGISRuntime.Toolkit.Controller
+
+import QtQuick
+import QtQml
+import QtQuick.Controls
+import QtQuick.Layouts
+
+/*!
+ \qmltype UtilityNetworkTrace
+ \inqmlmodule Esri.ArcGISRuntime.Toolkit
+ \ingroup ArcGISQtToolkitUiQmlViews
+ \since Esri.ArcGISRuntime 200.0
+ \brief The user interface for the UtilityNetworkTrace.
```

Comment:
This could use some beefed up doc. Maybe you could steal a few of the highlights from Nathan's blog, such as that the tool allows you to easily run a trace on a utility network with a trace configuration, the user workflow for how it works, etc

Reply:
Added more informative introduction to this tool.